### PR TITLE
Tweak debug to remove unreadable datapoints

### PR DESCRIPTION
**Pull request by sakridge**
#### Problem

Too much datapoint logging.

#### Summary of Changes

Lower level of some of the most common logs which are already duplicated with another human-readable print anyway.

Fixes #

### CODE REVIEW

**Comment by pgarg66** [comment]
The dashboard may not show some datapoints with this change.

**Comment by sakridge** [comment]
@pgarg66 do debug-level datapoints not get submitted? or they are just lower priority and may be dropped?

**Comment by pgarg66** [comment]
> @pgarg66 do debug-level datapoints not get submitted? or they are just lower priority and may be dropped?

It depends on the RUST_LOG level. We typically set RUST_LOG=info. So only datapoints with level >= info are getting submitted. Not sure if it has been changed recently, but that's how we started with it.

**Comment by sakridge** [comment]
> > @pgarg66 do debug-level datapoints not get submitted? or they are just lower priority and may be dropped?
> 
> It depends on the RUST_LOG level. We typically set RUST_LOG=info. So only datapoints with level >= info are getting submitted. Not sure if it has been changed recently, but that's how we started with it.

I see. Seems like we need to find some way to submit but not print them.

**Comment by pgarg66** [comment]
> > > @pgarg66 do debug-level datapoints not get submitted? or they are just lower priority and may be dropped?
> > 
> > 
> > It depends on the RUST_LOG level. We typically set RUST_LOG=info. So only datapoints with level >= info are getting submitted. Not sure if it has been changed recently, but that's how we started with it.
> 
> I see. Seems like we need to find some way to submit but not print them.

Yes, we have another datapoint, `vote-native` that needs the same facility. Currently it's spamming the logs. https://github.com/solana-labs/solana/issues/6057

**Comment by pgarg66** [comment]
@sakridge , @mvines and I talked about it earlier today. We can probably remove the log_level filter from datapoint submission. The datapoint logs will still use the filter, but submission will ignore the filter. Then we can adjust the datapoint levels independent of the log level.

File: metrics/src/datapoint.rs [hunk 1]
```diff
@@ -91,56 +91,40 @@ macro_rules! datapoint {
 #[macro_export]
 macro_rules! datapoint_error {
     ($name:expr) => {
-        if log::log_enabled!(log::Level::Error) {
-            $crate::submit($crate::datapoint!(@point $name), log::Level::Error);
-        }
+        $crate::submit($crate::datapoint!(@point $name), log::Level::Error);
```
Comment (sakridge) [inline metrics/src/datapoint.rs]: @pgarg66 that's what you're expecting right? Seems to work as expected.

**Comment by sakridge** [comment]
@mvines @pgarg66 Are we good with this PR?

File: metrics/src/datapoint.rs (hunk 1 above)
Comment (pgarg66) [reply metrics/src/datapoint.rs]: yes, that looks good

**Pull request merged**